src/modules/AuthenticationModule/components/Register/Register.tsx

```diff
@@ -1,10 +1,227 @@
+                                                    {...register("email",emailValidation)}
+                                                />
+                                            </div>
+                                            {errors.email && (
+                                                <p className="alert alert-danger">
+                                                    {errors.email.message}
+                                                </p>
+                                            )}
+
+                                            <div className="input-group pb-3">
+                                                <p className="my-1 register-text-color">Phone Number</p>
+                                                <input
+                                                    type="text"
+                                                    className="input-style"
+                                                    placeholder="Enter Your Phone Number"
+                                                    {...register("phoneNumber", phoneNumberValidation)}
+                                                />
+                                            </div>
+                                            {errors.phoneNumber && (
+                                                <p className="alert alert-danger">
+                                                    {errors.phoneNumber.message}
+                                                </p>
+                                            )}
+
+                                            <div className="input-group mb-3 position-relative">
+                                                <p className="my-1 register-text-color">
+                                                    Confirm Password
+                                                </p>
+                                                <input
+                                                    type={isConfirmPasswordShown ? "text" : "password"}
+                                                    className="input-style"
+                                                    placeholder="Confirm Your Password"
+                                                    {...register("confirmPassword", {
+                                                        required: "Confirm Password is required",
+                                                        validate: (value) =>
+                                                            value === watch("password") ||
+                                                            "Passwords do not match",
+                                                    })}
+                                                />
+                                                <div className="position-absolute end-0 px-2 py-4 mt-3">
+                                                    <span onClick={(e)=>handlePasswordVisibility(e,setisConfirmPasswordShown)}>
```

Comment:
This should be a button to make it accessible using the keyboard navigation (the tap key)